src/zenml/integrations/deepchecks/__init__.py

```diff
@@ -38,10 +38,13 @@ class DeepchecksIntegration(Integration):
     REQUIREMENTS = [
         "deepchecks[vision]==0.8.0",
         "torchvision>=0.14.0",
-        "pandas<2.0.0",
         "opencv-python==4.5.5.64",  # pin to same version
         "opencv-python-headless==4.5.5.64",  # pin to same version
         "tenacity!=8.4.0",  # https://github.com/jd/tenacity/issues/471
+        # Deepchecks materializer inherits from the PandasMaterializer, thus
```

Comment:
Why can't we actually import the pandas integration here then and put it's requirements?

Reply:
Should be theoretically possible. However, I am not 100% sure yet on how we will handle the PandasIntegration, so I will comeback to this.

Reply:
Btw, just as an additional note. `deepchecks` integration is a special case here. Much like our other integrations, [their requirements at 0.8.0 (the version that we use in ZenML)](https://github.com/deepchecks/deepchecks/blob/baea2cb313567fbd34d02c67980145e32ead4b15/requirements/requirements.txt#L1) actually allow us to use pandas v2. However, their code is trying to import [a function](https://github.com/deepchecks/deepchecks/blob/baea2cb313567fbd34d02c67980145e32ead4b15/deepchecks/utils/features.py#L21) that was eventually removed in `pandas>=2.1.0`, which leads to an error.

I will try to upgrade our `deepchecks` integration as the issue is resolved in their latest version.